### PR TITLE
Add /interface_locales path from RD-17485

### PR DESCRIPTION
**Pull request by brenauj**
This PR adds doc about the new route to get user interface locales list.

### CODE REVIEW

File: specs/engage-digital_openapi3.yaml [hunk 1]
```diff
@@ -2081,6 +2081,22 @@ paths:
...
+      operationId: getAllUiLocales
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                items:
+                  $ref: '#/components/schemas/UiLocale'
+                type: array
+          description: List of all locales
```
Comment (CedricBm) [inline specs/engage-digital_openapi3.yaml:2096]: No need to add "UI" in this description too? (and "content" in the other one)

**Review by CedricBm — COMMENTED** [review]
Just a minor comment, lgtm otherwise

File: specs/engage-digital_openapi3.yaml (hunk 1 above)
Comment (CedricBm) [reply specs/engage-digital_openapi3.yaml]: Running swaggman again doesn't update the postman?

Comment (brenauj) [reply specs/engage-digital_openapi3.yaml]: I added it

Comment (CedricBm) [reply specs/engage-digital_openapi3.yaml]: ok, I guess it's a no for my second question :)

**Comment by CedricBm** [comment]
LGTM once the related code is deployed.

**Pull request merged**